Formula/z3.rb

```diff
@@ -21,10 +13,19 @@ class Z3 < Formula
     sha256 "81c89c70da771d0d1faae38657e43849f741cb68486afbf50a21be76bf612799" => :mavericks
   end
 
+  option "without-python", "Build without python 2 support"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on :python3 => :optional
+
+  if build.without?("python3") && build.without?("python")
+    odie "z3: --with-python3 must be specified when using --without-python"
+  end
+
   def install
-    inreplace "scripts/mk_util.py", "dist-packages", "site-packages"
+    # This inreplace can be removed on next release.
+    inreplace "scripts/mk_util.py", "dist-packages", "site-packages" if build.stable?
 
-    Language::Python.each_python(build) do |python, version|
+    Language::Python.each_python(build) do |python, _|
```

Comment:
If `version` is not used here, you can replace it with `_version`. This will silence the Ruby syntax warning about an unused variable, but it will remain clear what that second argument is for.